exporters/variables-scss/src/generators/cssObjectGenerator.ts

```diff
@@ -44,36 +44,74 @@ export const getTokenAlias = (token: Token): string => {
   return alias;
 };
 
+const formatTypographyName = (tokenNameParts: string[]): string => {
```

Comment:
I'd rather move it to a helper file (probably into cssObjectHelper)

Reply:
it was in my plan to do, but i forget xD thanks